week-1-ts-ecosystem/w01e02-enter-the-world-of-typescript/webpack.development.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+    mode: "development",
```

Comment:
Twoja konfiguracja dla `dev` i dla `prod` różni się tylko w niewielkim stopniu, tutaj jest to tylko ta flaga.

To warto skorzystać z webpack merge - żeby się nie powtarzać.
Zobacz _doc_:
```
https://webpack.js.org/guides/production/
```